Assignment/solution_student/index.html

```diff
@@ -4,30 +4,83 @@
     <meta charset="UTF-8">
     <title>Relaxr Blog</title>
     <meta name="description" content="An HTML Starter Template">
-    
-    <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
-    <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
-
-    <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" type="image/ico" href="images/favicon.png">
   </head>
   <body>
-
-    <!-- YOUR MARKUP GOES HERE! -->
-    <div class="container">
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+      <header class="clearfix">
+        <h1>Relaxr</h1>
+      <nav>
+       <ul>
+        <li><a href="#" class="nav-links">About</a></li>
+        <li><a href="#" class="nav-links">FAQ</a></li>
+        <li><a href="#" class="nav-links">Team</a></li>
+        <li><a href="#" class="nav-links">Contact Us</a></li>
+        <li><a href="#"class="nav-links">Blog</a></li>
+       </ul> 
+      </nav>
+     </header> 
+
+    <div class="main clearfix">
+      <div class="main-content">
+
+      <h2> How I implemented Relaxr in 2 weeks and changed my life.</h2>
```

Comment:
Because this is a list of multiple articles, they should be contained individually somehow, and what better way to do that then wrap them each in an `<article>` tag? :)